test/helpers/vagrant.go

```diff
@@ -75,11 +75,16 @@ func CreateVM(scope string) error {
 // for the provided Vagrant of name vmName. Returns an error if
 // `vagrant ssh-config` fails to execute.
 func GetVagrantSSHMetadata(vmName string) ([]byte, error) {
-	cmd := getCmd(fmt.Sprintf("vagrant ssh-config %s", vmName))
+	cmd := getCmd(fmt.Sprintf("vagrant global-status | grep %s | awk 'NR==1{ print $1 }'", vmName))
```

Comment:
I presume this change is due to the fact that there is a `Vagrantfile` in the `test` directory as well as at the root of the Cilium repository? This is fine for now. We should just coalesce the Vagrantfiles so that all Vagrant VMs used for developer purposes are located in one file. This can be done separately outside of this issue, though.